tests/components/humidifier/test_intent.py

```diff
@@ -207,15 +206,14 @@ async def test_intent_set_unknown_mode(
     mode_calls = async_mock_service(hass, DOMAIN, SERVICE_SET_MODE)
     await intent.async_setup_intents(hass)
 
-    try:
+    with pytest.raises(
+        IntentHandleError, match="Entity bedroom humidifier does not support eco mode"
```

Comment:
This is not the same. The previous version was an exact match, this one isn't. Match is an expression.